bin/add_submitter_annotations.R

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env Rscript
+
+# This script adds submitter annotations, if provided, to the unfiltered SCE's colData.
+# If the file was not provided, this script has no effect.
+
+# import libraries
+suppressPackageStartupMessages({
+  library(optparse)
+  library(SingleCellExperiment)
+})
+# set up arguments
+option_list <- list(
+  make_option(
+    opt_str = c("-u", "--unfiltered_file"),
+    type = "character",
+    help = "path to output unfiltered rds file. Must end in .rds"
+  ),
```

Comment:
It probably doesn't matter much, but I don;t think there is anything in this script that makes it specific to unfiltered SCEs. 

I would make the input name more general! 
```suggestion
  make_option(
    opt_str = c("-f", "--sce_file"),
    type = "character",
    help = "path to SingleCellExperiment file to update. Must end in .rds"
  ),
```